capstone/src/main/java/com/google/univiz/api/MapData.java

```diff
@@ -1 +1,19 @@
 package com.google.univiz.api;
+
+/*
```

Comment:
nit - this public facing documentation should be javadoc.
```suggestion
/**
```